manifest.json

```diff
@@ -1,29 +1,35 @@
 {
-  "name": "SQL Importer",
-  "description": "Import SQL data from your own database",
+  "name": "SQL Connector",
+  "description": "Import data from your SQL databases",
   "picture": "picture.png",
-  "version": "0.2.3",
+  "version": "0.3.0",
   "tags": [
     "incoming",
     "oneColumn"
   ],
   "private_settings": [
     {
+      "type": "string",
+      "format": "title",
+      "name": "connector_info",
+      "title": "Connector Configuration"
+    },
+    {
       "name": "enabled",
-      "title": "Enable Sync",
+      "title": "Enable Sync (default interval is 3 hours)",
       "type": "boolean",
       "default": "false"
     },
     {
       "name": "overwrite",
-      "title": "Overwrite data in Hull in case of a conflict",
+      "title": "Always overwrite data in Hull with values from SQL",
       "type": "boolean",
-      "default": "true"
+      "default": "false"
```

Comment:
@unity I like the idea of making it positive. "Overwrite" raises some sort of danger flag in my head... How about we label it "Use SQL value in case of data conflict"? That way we can leave the flag set to `true` which is in alignment with the expected standard behavior and it is explained in the instructions with an example what a data conflict is, so there should be no confusion for the user here...